sunpy/net/hek/utils.py

```diff
@@ -0,0 +1,242 @@
+import os
+import re
+import json
+import warnings
+from pathlib import Path
+
+import numpy.ma
+from regions import PolygonSkyRegion
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.time import parse_time
+
+__all__ = [
+    'freeze',
+    'parse_times',
+    'parse_values_to_quantities',
+    'UNIT_FILE_PATH',
+    'COORD_FILE_PATH'
+]
+
+UNIT_FILE_PATH = Path(os.path.dirname(__file__)) / "unit_properties.json"
+COORD_FILE_PATH = Path(os.path.dirname(__file__)) / "coord_properties.json"
+
+def freeze(obj):
+    """
+    Create a hashable representation of a dict or list.
+    Otherwise returns the original input.
+    """
+    if isinstance(obj, dict):
+        return tuple((k, freeze(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return tuple(freeze(elem) for elem in obj)
+    return obj
+
+# NOTE: Needs unit test
```

Comment:
This still true?

Reply:
`parse_times` has a unit test, but it needs to be written in `test_utils` instead of `test_hek`.